### PR TITLE
Theming from pals houndci

### PR DESCRIPTION
**Pull request by erik-engstrom**
Fixes theming-from-pals Hound CI errors on #1727 

Fixes linting errors identified by Hound CI in .scss files 

@samvera/hyku-code-reviewers

### CODE REVIEW

**Comment by erik-engstrom** [comment]
This should fix all of the fixable HoundCI errors.

File: app/assets/stylesheets/themes/cultural_repository.scss [hunk 1]
```diff
@@ -1,53 +1,68 @@
-.cultural_repository  {
+.cultural__repository {
```
Comment (labradford) [inline app/assets/stylesheets/themes/cultural_repository.scss]: I get that the linter told you to change this but you need to leave it as cultural_repository. This selector is generated by a controller.

File: app/assets/stylesheets/themes/cultural_repository.scss [hunk 2]
```diff
@@ -1,53 +1,68 @@
...
   // Note: copied over to override /* line 5, /home/app/webapp/app/assets/stylesheets/hyku.scss */
   #masthead #logo img {
     max-height: 100px !important;
   }
+
   $navbar-height: 100px !important;
   ////// End Navbar //////
 
-  div#featured_works > div.featured-works-4-column-layout:nth-child(1n+5){ display: none !important; }
+  div#featured__works > div.featured-works-4-column-layout:nth-child(1n+5) {
```
Comment (labradford) [inline app/assets/stylesheets/themes/cultural_repository.scss]: did you mean to change this to use the double underscore?

**Review by labradford — CHANGES_REQUESTED** [review]
one small change requested

File: app/assets/stylesheets/themes/cultural_repository.scss (hunk 2 above)
Comment (erik-engstrom) [reply app/assets/stylesheets/themes/cultural_repository.scss]: There was a Block Element Modifier (BEM) error on this one that Dylan and I worked on to come up with a solution. We found that the format was requesting a double underscore to pass. Should I change it back to a single underscore?

Comment (aprilrieger) [reply app/assets/stylesheets/themes/cultural_repository.scss]: Yes that would be great Erik, thank you!

Comment (erik-engstrom) [reply app/assets/stylesheets/themes/cultural_repository.scss]: Not a problem! Double underscore removed to a single underscore and the code has been pushed up for review.

**Review by labradford — APPROVED** [review]

**Pull request merged**